includes/class-wc-payments-utils.php

```diff
@@ -887,4 +887,47 @@ public static function get_wpcore_request_class(): string {
 			? '\\WpOrg\\Requests\\Requests'
 			: '\\Requests';
 	}
+
+	/**
+	 * Returns a merchant-friendly description of the dispute reason.
+	 *
+	 * This mapping is duplicated in client/disputes/strings.ts and on Server.
+	 *
+	 * @param string $reason The dispute reason.
+	 *
+	 * @return string
+	 */
+	public static function get_dispute_reason_description( string $reason ): string {
+		switch ( $reason ) {
+			case 'bank_cannot_process':
+				return __( 'Bank can not process', 'woocommerce-payments' );
+			case 'check_returned':
+				return __( 'Check returned', 'woocommerce-payments' );
+			case 'credit_not_processed':
+				return __( 'Credit not processed', 'woocommerce-payments' );
+			case 'customer_initiated':
+				return __( 'Customer initiated', 'woocommerce-payments' );
+			case 'debit_not_authorized':
+				return __( 'Debit not authorized', 'woocommerce-payments' );
+			case 'duplicate':
+				return __( 'Duplicate', 'woocommerce-payments' );
+			case 'fraudulent':
+				return __( 'Transaction unauthorized', 'woocommerce-payments' );
```

Comment:
To keep this PR changelog correct, should this be `Fraudulent` to remain consistent with `client/disputes/strings.ts`?

https://github.com/Automattic/woocommerce-payments/blob/7cef6df74173a4d6e278f08ec4346b588e48bb5d/client/disputes/strings.ts#L112

I see that this is being changed to `Transaction unauthorized` in #6472, so perhaps that PR could handle the change across both files.

```suggestion
				return __( 'Fraudulent', 'woocommerce-payments' );
```

Reply:
Good thinking, I'll update the other PR